### PR TITLE
fix(docs): remove docsify tabs

### LINKED ISSUE #51958: Remove docsify tabs from docs
### Describe the Issue

**ISSUE :** On the Contribution Guide Page, there is a UI issue. Tab text for unselected tabs is not displayed. Here is the screenshot:

### Affected Page

https://contribute.freecodecamp.org

### Steps to Reproduce

**Visit these pages -** 
https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=configuring-dependencies
https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=step-3-start-mongodb-and-seed-the-database

Other pages using this _Common Component_ may also experience this issue. Consider checking those pages as well.

### Expected behavior

The text must appear for Tab either it is selected or not

### Screenshots

image

### System

- Device: Laptop
- OS: Windows 10 Version 22H2
- Browser: Chrome 118

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by lakshmi930**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #51958

### CODE REVIEW

**Review by gikf — CHANGES_REQUESTED** [review]
Hey @lakshmi930,

Thank you for opening this pull request.

This isn't what should be done to resolve the linked issue. See https://github.com/freeCodeCamp/freeCodeCamp/issues/51958#issuecomment-1765691688.

Thank you and happy coding.

**Comment by lakshmi930** [comment]
@gikf Thank you for tagging the comment here, had missed that comment. I have updated the PR again just to be sure if this is what is required to be done and if my understanding is correct. I can then continue on other pages if it's the correct approach.

**Comment by ieahleen** [comment]
doing this, the docsify-tabs plugin is no longer needed, can you remove it and its costumization from the html file (`./docs/index.html`)? see my comment in https://github.com/freeCodeCamp/freeCodeCamp/issues/51958#issuecomment-1763036345

**Review by gikf — COMMENTED** [review]
There are two more files that contain tabs, with Windows specific instructions:

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/docs/troubleshooting-development-issues.md
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/docs/how-to-setup-freecodecamp-mobile-app-locally.md

**Comment by lakshmi930** [comment]
Yup, just wanted to confirm if I'm doing the right thing. Have removed the tab occurrences from the main docs and other languages @gikf 
Removed the plugin and the css customization as well @ieahleen 
Is there any tests or anything else also that I need to check here?

**Comment by gikf** [comment]
Hey @lakshmi930,

as the bot mentioned, only files from English language should be changed. Updating other languages will be handled using the translation workflow.

**Review by naomi-lgbt — APPROVED** [review]

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md [hunk 1]
```diff
@@ -138,22 +138,10 @@ The default API keys and environment variables are stored in the file `sample.en
...
 ```
 
-#### **Windows**
-
-```console
-copy sample.env .env
-```
-
-<!-- tabs:end -->
-
```
Comment (ieahleen) [inline docs/how-to-setup-freecodecamp-mobile-app-locally.md:156]: This is not needed to remove, the mobile app support Windows development. Please remove only the opening and closing of the tabs, not the windows content here

File: docs/index.html [hunk 2]
```diff
@@ -262,8 +262,6 @@
     </script>
     <!-- docsify (latest v4.x.x)-->
     <script src="https://cdn.jsdelivr.net/npm/docsify@4/lib/docsify.min.js"></script>
-    <!-- docsify-tabs (latest v1.x.x) -->
-    <script src="https://cdn.jsdelivr.net/npm/docsify-tabs@1"></script>
```
Comment (ieahleen) [inline docs/index.html:266]: for this file there is to remove the customization too, lines 159-165

https://github.com/freeCodeCamp/freeCodeCamp/blob/d0e816207b2d9e2f3fa2b8e0eda10edf4dcbfe3e/docs/index.html#L159-L165

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
We are almost there, I have a couple of things to say

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md (hunk 1 above)
Comment (lakshmi930) [reply docs/how-to-setup-freecodecamp-mobile-app-locally.md:156]: Sure, updating

File: docs/index.html (hunk 2 above)
Comment (lakshmi930) [reply docs/index.html:266]: image
Is this okay so there's no conflict in headings?

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md (hunk 1 above)
Comment (lakshmi930) [reply docs/how-to-setup-freecodecamp-mobile-app-locally.md:156]: image
 Is this okay so there's no conflict in headings?

File: docs/index.html (hunk 2 above)
Comment (lakshmi930) [reply docs/index.html:266]: Removed

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md (hunk 1 above)
Comment (ieahleen) [reply docs/how-to-setup-freecodecamp-mobile-app-locally.md]: No, it's good that the headings are at the same level

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md [hunk 3]
```diff
@@ -138,22 +138,16 @@ The default API keys and environment variables are stored in the file `sample.en
 # Populate it with the necessary API keys and secrets:
 ```
 
-<!-- tabs:start -->
-
-#### **macOS/Linux**
-
+macOS/Linux:
```
Comment (ieahleen) [inline docs/how-to-setup-freecodecamp-mobile-app-locally.md:141]:
```suggestion
#### *macOS/Linux*

```

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md [hunk 4]
```diff
@@ -138,22 +138,16 @@ The default API keys and environment variables are stored in the file `sample.en
...
 ```console
 cp sample.env .env
 ```
 
-#### **Windows**
-
+Windows:
```
Comment (ieahleen) [inline docs/how-to-setup-freecodecamp-mobile-app-locally.md:146]:
```suggestion
#### **Windows**

```

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md [hunk 5]
```diff
@@ -417,21 +405,15 @@ copy sample.env .env
...
-```console
-cp ./shared/config/curriculum.json ../mobile/mobile-app/curriculum.json
-```
-
-#### **Windows**
-
-```console
-copy .\config\curriculum.json ..\mobile\mobile-app\curriculum.json
-```
+    macOS/Linux:
```
Comment (ieahleen) [inline docs/how-to-setup-freecodecamp-mobile-app-locally.md:408]:
```suggestion
#### **macOS/Linux**

```

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md [hunk 6]
```diff
@@ -417,21 +405,15 @@ copy sample.env .env
...
+    ```console
+    cp ./shared/config/curriculum.json ../mobile/mobile-app/curriculum.json
+    ```
 
-<!-- tabs:end -->
+    Windows:
```
Comment (ieahleen) [inline docs/how-to-setup-freecodecamp-mobile-app-locally.md:413]:
```suggestion
#### **Windows**

```

File: docs/troubleshooting-development-issues.md [hunk 7]
```diff
@@ -51,27 +51,12 @@ git clean -ifdX
 
 If you can't sign in, and instead you see a banner with an error message that it will be reported to freeCodeCamp, please double-check that your local port `3000` is not in use by a different program.
 
-<!-- tabs:start -->
-
-#### **macOS/Linux/WSL on Windows - From Terminal:**
```
Comment (ieahleen) [inline docs/troubleshooting-development-issues.md:56]: Leave the indication From Terminal here please

```
#### **From Terminal:**

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md [hunk 8]
```diff
@@ -387,21 +381,15 @@ Follow these steps:
...
-```console
-cp sample.env .env
-```
-
-#### **Windows**
-
-```console
-copy sample.env .env
-```
+    macOS/Linux:
```
Comment (ieahleen) [inline docs/how-to-setup-freecodecamp-mobile-app-locally.md:384]:
```suggestion
#### **macOS/Linux**

```

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md [hunk 9]
```diff
@@ -387,21 +381,15 @@ Follow these steps:
...
+    ```console
+    cp sample.env .env
+    ```
 
-<!-- tabs:end -->
+    Windows:
```
Comment (ieahleen) [inline docs/how-to-setup-freecodecamp-mobile-app-locally.md:389]:
```suggestion
#### **Windows**

```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]

File: docs/how-to-setup-freecodecamp-mobile-app-locally.md (hunk 1 above)
Comment (lakshmi930) [reply docs/how-to-setup-freecodecamp-mobile-app-locally.md]: Cool, I have added them back based on what you have suggested. Can you check them again please?

File: docs/troubleshooting-development-issues.md (hunk 7 above)
Comment (lakshmi930) [reply docs/troubleshooting-development-issues.md:56]: Sure, done 👍🏼

**Review by majestic-owl448 — APPROVED** [review]

**Review by gikf — APPROVED** [review]

**Pull request merged**